39/documentation.html

```diff
@@ -30,7 +30,7 @@
     <!--#include virtual="toc.html" -->
   </div>
   <div class="right">
-    <!--#include virtual="../includes/_docs_banner.htm" -->
+    <!--//#include virtual="../includes/_docs_banner.htm" -->
```

Comment:
What's this change for?

Reply:
Interesting...
I don't change these manually. 🤔